packages/device-id/package.json

```
@@ -23,14 +23,19 @@
   ],
   "license": "Apache-2.0",
   "main": "dist/index.js",
+  "types": "./dist/index.d.ts",
   "exports": {
     ".": {
-      "require": "./dist/index.js",
-      "import": "./dist/.esm-wrapper.mjs",
-      "types": "./dist/index.d.ts"
+      "require": {
+        "default": "./dist/index.js",
+        "types": "./dist/index.d.ts"
+      },
```

A "types" condition like this is redundant, since TSC will automatically infer it from the .js file.

```suggestion
      "require": "./dist/index.js",
```

---

so just for the "require" ones? I left a comment elsewhere where I already had a suspicion like that.